slick/slick.js

```diff
@@ -1273,7 +1272,7 @@ Issues: http://github.com/kenwheeler/slick/issues
 		}
 
 		if (creation) {
-			_.$slider.trigger('init', [_]);
+			_.triggerEvent(_.$slider[0], 'init', [_]);
```

Comment:
It would be nice to have a standard ... either get() (to have an Array), get(0), (to have first of array)... 

cuz _.$slider[0] could be an array or an element, there's no way of telling it with only just this line of code.